09-secret-management-and-ingress-controller.md

```diff
@@ -7,8 +7,8 @@ Previously you have configured [workload prerequisites](./08-workload-prerequisi
 1. Get the AKS Ingress Controller Managed Identity details.
 
    ```bash
-   INGRESS_CONTROLLER_WORKLOAD_IDENTITY_CLIENT_ID=$(az deployment group show --resource-group rg-bu0001a0008 -n cluster-stamp --query properties.outputs.aksIngressControllerPodManagedIdentityClientId.value -o tsv)
-   echo INGRESS_CONTROLLER_WORKLOAD_IDENTITY_CLIENT_ID: $INGRESS_CONTROLLER_WORKLOAD_IDENTITY_CLIENT_ID
+   export INGRESS_CONTROLLER_WORKLOAD_IDENTITY_CLIENT_ID_AKS_BASELINE=$(az deployment group show --resource-group rg-bu0001a0008 -n cluster-stamp --query properties.outputs.aksIngressControllerPodManagedIdentityClientId.value -o tsv)
```

Comment:
Why are we exporting now?

Reply:
because https://github.com/mspnp/aks-baseline/blob/main/saveenv.sh save the variables that are exported and end by _AKS_BASELINE. 
Without the export, the variables are not saved. 
That is good when you for some reason need to re-start a deploy process in the middle, you have the variable values till the moment you executed the latest  saveenv.sh.